src/components/DraggableList/index.native.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import DraggableFlatList from 'react-native-draggable-flatlist';
+import {FlatList} from 'react-native-gesture-handler';
+import type {DraggableListProps, DraggableListType} from './types';
+
+function DraggableList<T>({renderClone, shouldUsePortal, ...viewProps}: DraggableListProps<T>, ref: React.ForwardedRef<FlatList<T>>) {
+    return (
+        <DraggableFlatList
+            ref={ref}
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...viewProps}
+        />
+    );
+}
+
+DraggableList.displayName = 'DraggableList';
+
+// We have to assert the type here because we use generic forwrded ref
```

Comment:
```suggestion
// We have to assert the type here because we use generic forwarded ref
```

Reply:
Hey, I've removed this comment, as I used [this new approach](https://expensify.slack.com/archives/C01GTK53T8Q/p1695110739750909).

cc @blazejkustra